src/plugins/data/public/ui/typeahead/suggestions_component.tsx

```diff
@@ -56,8 +100,8 @@ export class SuggestionsComponent extends Component<Props> {
 
     return (
       <div className="reactSuggestionTypeahead">
-        <div className="kbnTypeahead">
-          <div className="kbnTypeahead__popover">
+        <div className="kbnTypeahead" ref={this.kbnTypeaheadDivRefInstance}>
+          <div className="kbnTypeahead__popover" ref={this.kbnTypeaheadPopoverDivRefInstance}>
```

Comment:
not sure that we need two refs. Looks like all works fine after removing kbnTypeaheadPopoverDivRefInstance

all what you need it's just a modify a little styles


 ```
 .kbnTypeahead__popover {
    max-height: inherit;
    @include euiBottomShadow($adjustBorders: true);
    border: 1px solid;
    border-color: $euiBorderColor;
    color: $euiTextColor;
    background-color: $euiColorEmptyShade;
    position: relative;
    top: -2px;
    z-index: $euiZContentMenu;
    width: 100%;
    border-bottom-left-radius: $euiBorderRadius;
    border-bottom-right-radius: $euiBorderRadius;
    overflow-y: auto;

    .kbnTypeahead__items {
      @include euiScrollBar;
    }
```